src/css/_city-details.scss

```
@@ -0,0 +1,40 @@
+.city-details-popup {
+  position: absolute;
+  z-index: 1002;
+  background-color: white;
+  border: 1px solid gray;
+  width: 40%;
+  height: 60%;
+
+  @media only screen and (min-width: 90em) {
+    height: auto;
+  }
+
+  top: 4rem;
+  right: 1rem;
+  overflow: auto;
+  display: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
```

Sass can rewrite this to:

```suggestion
a {
  text-decoration: none;

  &:hover {
    text-decoration: underline;
  }
}
```

I personally like this because it groups related things better.